backend_uml.go

```diff
@@ -0,0 +1,200 @@
+// +build linux
+// +build amd64
+
+package fakemachine
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+)
+
+type umlBackend struct {
+	machine         Machine
+	kernelPath      string
+	moduleDir       string
+	slirpHelperPath string
+}
+
+func newUmlBackend(m Machine) umlBackend {
+	b := umlBackend{}
+	b.machine = m
+
+	b.kernelPath = "/usr/bin/linux.uml"
+	b.moduleDir = "/usr/lib/uml/modules"
+	b.slirpHelperPath = "/usr/bin/slirp-helper"
+
+	// TODO remove chris debug paths
+	base := "/home/obbardc/projects/debos/uml/"
+	b.kernelPath = base + "linux-uml/linux"
+	b.moduleDir = base + "linux-uml/modules/lib/modules"
+	b.slirpHelperPath = base + "libslirp-rs/target/debug/libslirp-helper"
```

Comment:
as things are getting close; drop these. Also for the binaries doesn't go have helpers to search in ones PATH ?